src/main/java/nextstep/courses/domain/lectures/FreeLecture.java

```diff
@@ -48,35 +70,53 @@ public boolean isFree() {
   }
 
   @Override
-  public boolean recruiting() {
-    return LectureStatus.RECRUITING.equals(this.lectureStatus);
+  public boolean isRecruiting() {
+    return LectureRecruitingStatus.RECRUITING.equals(this.lectureRecruitingStatus);
```

Comment:
```suggestion
    return lectureRecruitingStatus.isRecruiting();
```
enum 또한  객체라 메시지를 보내 구현 가능함